test/appium/tests/medium/test_pairing_devices_sync_m.py

```diff
@@ -63,15 +97,124 @@ def prepare_devices(self):
+
+    @marks.testrail_id(702288)
+    def test_pairing_sync_currency(self):
+        self.wallet_2 = self.home_2.wallet_button.click()
+        self.wallet_2.just_fyi('Check currency is set to AFN on device 2 after sync')
+        if not self.wallet_2.element_by_text_part('AFN').is_element_displayed():
+            self.errors.append('Currency is not set to AFN on device 2 after sync')
+
+        self.wallet_2.just_fyi('Change currency to AED on device 2')
+        currency_aed = 'Emirati Dirham (AED)'
+        self.wallet_2.set_currency(currency_aed)
+        self.wallet_2.navigate_up_button.click()
+
+        self.wallet_1 = self.home_1.wallet_button.click()
+        self.wallet_1.just_fyi('Check currency set to AED on device 1 after changing value on paired device 2')
+        if not self.wallet_1.element_by_text_part('AED').is_element_displayed():
+            self.errors.append('Currency is not set to AED on device 1 after value was set on paired device 2')
+
+        self.device_3.put_app_to_background_and_back()
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702288)
+    def test_pairing_sync_watch_only_account(self):
+        self.wallet_2 = self.home_2.wallet_button.click()
+        account_button = self.wallet_2.get_account_by_name(self.account_name)
+        if not account_button.is_element_displayed():
+            self.errors.append('Watch only account is displayed on device 2 after sync')
+
+        self.wallet_2.just_fyi('Add another watch only wallet by paired device 2')
+        if not self.wallet_2.add_account_button.is_element_displayed(3):
+            self.wallet_2.accounts_status_account.swipe_left_on_element()
+        self.wallet_2.add_account_button.click()
+        self.wallet_2.add_watch_only_address_button.click()
+        self.wallet_2.enter_address_input.send_keys(ens_user['address'])
+        account_name_2 = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
+        self.wallet_2.account_name_input.send_keys(account_name_2)
+        self.wallet_2.add_account_generate_account_button.click()
+
+        self.wallet_1.just_fyi('Check newly added watch only account appeared on paired device 1')
+        self.wallet_1 = self.wallet_1.wallet_button.click()
+        account_button_2 = self.wallet_1.get_account_by_name(account_name_2)
+        if not account_button_2.is_element_displayed():
+            self.wallet_1.accounts_status_account.swipe_left_on_element()
+            if not account_button_2.is_element_displayed():
+                self.errors.append('New watch only account has not appeared on paired device 1')
+
+        self.errors.verify_no_errors()
 
     @marks.testrail_id(702269)
     @marks.xfail(reason="too long setup, can fail with Remote end closed connection")
     def test_pairing_sync_initial_community_send_message(self):
-        # Pricture sync is not implemented yet
+        # Picture sync is not implemented yet
```

Comment:
May delete prerequisite for this part and this part from this e2e, as it will be in critical.

Reply:
+